bin/docker-php

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env bash
```

Comment:
For the other Docker images, we named the script based on the name of the image, so `php` instead of `docker-php` !